controllers/hibp.js

```diff
@@ -24,18 +24,21 @@ function getAddressesAndLanguageForEmail(recipient) {
         recipientEmail: recipient.primary_email,
         breachedEmail: recipient.email,
         signupLanguage,
+        preFxaSubscriber: false,
```

Comment:
@groovecoder Is there a better way/place to check this? I'm not totally familiar with all of this snazziness yet and am kind of winging it here.

Reply:
In general, if `recipient.fxa_uid` is null/empty, then they are a pre-FxA record. So, we could move this check up outside of these `if` conditionals ...

```
function getAddressesAndLanguageForEmail(recipient) {
  const signupLanguage = recipient.signup_language;
  const preFxaSubscriber = !recipient.fxa_uid;